payments/credit-card/index.md

```diff
@@ -56,7 +56,9 @@ receive the state of the transaction.
 You will redirect the payer to Swedbank Pay hosted pages to collect the credit
 card information.
 
-![Screnshot-1]
+
+![Screenshot-1]
```

Comment:
Can we get a nice name, alt text and dimensions for the screenshots? It should look something like this:

```suggestion
![Card Payment][card-payment]{:width="x" height="y"}
```